mp/src/materialsystem/stdshaders/fxctmp9/neo_motion_model_vs20.inc

```diff
@@ -0,0 +1,160 @@
+#include "shaderlib/cshader.h"
+class neo_motion_model_vs20_Static_Index
+{
+public:
+	neo_motion_model_vs20_Static_Index( )
+	{
+	}
+	int GetIndex()
+	{
+		// Asserts to make sure that we aren't using any skipped combinations.
+		// Asserts to make sure that we are setting all of the combination vars.
+#ifdef _DEBUG
+#endif // _DEBUG
```

Comment:
Nit: either missing the asserts mentioned, or the comments are outdated

Reply:
I took the example model include file and just did a replace for the word example